### PR TITLE
trezor: do not rely on deprecated ckd_public

### PR DESCRIPTION
**Pull request by matejcik**
electrum has its own xpub parsing method; use that in favor of Trezor's which is now considered private and moved away

### CODE REVIEW

File: plugins/trezor/trezor.py [hunk 1]
```diff
@@ -268,6 +266,12 @@ def _initialize_device(self, settings, method, device_id, wizard, handler):
             client.load_device_by_xprv(item, pin, passphrase_protection,
                                        label, language)
 
+    def _make_node_path(self, xpub, address_n):
+        xtype, depth, fingerprint, child_num, chain_code, key = bitcoin.deserialize_xpub(xpub)
```
Comment (SomberNight) [inline plugins/trezor/trezor.py]: you need
```python
fingerprint = struct.unpack('>I', fingerprint)[0]
child_num   = struct.unpack('>I', child_num)[0]
```

Comment (SomberNight) [inline plugins/trezor/trezor.py]: `bitcoin` is not imported

**Comment by SomberNight** [comment]
In multisig wallets that involve a trezor cosigner.

**Comment by matejcik** [comment]
this seems to work then

**Comment by SomberNight** [comment]
Great; thanks.

**Pull request merged**